examples/worlds/tunnel.sdf

```diff
@@ -33,17 +177,36 @@
     <include>
       <static>true</static>
       <name>BaseStation</name>
```

Comment:
just a suggestion if we want to keep this world consistent with subt: we now call this staging_area in subt and have another base_station model that sits near the back of the staging area:
https://app.ignitionrobotics.org/OpenRobotics/fuel/models/Base%20Station

Reply:
There's no need to keep consistency with SubT, it's just a coincidence that some models were pulled from there. But I updated the name in any case, thanks.